### DISCUSSION TITLE
.select('*').subscribe() help please

### DISCUSSION THREAD

**Discussion by emmyduruc** [supabase/supabase#12812] [General]
can anyone help please i am new to using supabase and i am trying to connect to the real-time database i have toggled all the button (replication) yet i get an error when i call the method subscribe but without subscribe everything works fine.
 
 Project: React-Native
 
 const {data: continents, error} = await database
      .from('continents')
      .select('*').subscribe();
      
   got this error  Property 'subscribe' does not exist on type 'PostgrestFilterBuilder<any, any, { [x: string]: any; }[]>'.ts(2339)

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/reference/javascript/subscribe
